src/db/PatientEntity.ts

```diff
@@ -0,0 +1,10 @@
+import mongoose, { Schema, Types } from 'mongoose'
+import Patient from '../model/Patient'
+
+const PatientSchema = new Schema({
```

Comment:
Let's use json schema for the schemas so we can add them to fastify too when needed.